process/peer/process.go

```diff
@@ -75,6 +75,7 @@ type validatorStatistics struct {
 	jailedEnableEpoch               uint32
 	belowSignedThresholdEnableEpoch uint32
 	flagJailedEnabled               atomic.Flag
+	flagInactiveToJailCorrection    atomic.Flag
```

Comment:
flagJailedEnabled is not used anymore

Reply:
it is used in PeerAccountToValidatorInfo

Reply:
ok